www/generate_benchmark_plot.js

```diff
@@ -0,0 +1,116 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: William Haack (will@cockroachlabs.com)
+
+function getTestDataFromJSON(directory, test, callback) {
+    fileName = directory + '/' + test + '.json'
+    loadJSON(fileName, callback);
+}
+
+function loadTestListFromJSON(callback) {
+    loadJSON('test_names.json', callback);
+}
+
+function getResultsFromTestData(dates, testData) {
+    var results = {
```

Comment:
Could be simplified using lodash, ES6 built-ins, and/or jquery

Here's a start
```
dates.map((d)=>testData(d)).reduce((memo, curr)=>{
  $.each(memo, (v, k)=>{memo[k] = v + curr[k];});
  return memo;
})